usage.js

```diff
@@ -9,14 +9,15 @@
 // Use this in "module" type nodejs projects
 import { config as loadEnv } from 'dotenv';
 import { SDK, Auth, TEMPLATES } from './index.js';
+import { ethers } from 'ethers';
 loadEnv();
 
 const acc = new Auth({
   privateKey: process.env.WALLET_PRIVATE_KEY,
   projectId: process.env.INFURA_PROJECT_ID,
   secretId: process.env.INFURA_PROJECT_SECRET,
-  rpcUrl: process.env.EVM_RPC_URL,
-  chainId: 4,
+  rpcUrl: 'https://polygon-mainnet.g.alchemy.com/v2/cyKpdJC2AHsuKa_NfKJ21rS-qmM2bQwP',
```

Comment:
we're not going to publicly publish an "infura-sdk" that uses an "alchemy" endpoint right? Is it because you were not able to get your infura account with polygon add-on enable? if that's the case, it can be solved

Reply:
lol sorry!!! i was trying to see if alchmey threw the same error 😂  (it does!)